pkg/genericapiserver/genericapiserver.go

```diff
@@ -768,7 +768,7 @@ func shouldGenSelfSignedCerts(certPath, keyPath string) bool {
 	return true
 }
 
-func (s *GenericAPIServer) installAPIGroup(apiGroupInfo *APIGroupInfo) error {
+func (s *GenericAPIServer) InstallAPIGroup(apiGroupInfo *APIGroupInfo) error {
```

Comment:
Let this be private.
InstallAPIGroups is a public function which everyone should be calling.